parallaxinc/Parallax-IDE
# Syntax

#### What's this PR do?

Apply syntax colors as described in issue #71 
Targets PBASIC full syntax 
Closes #85 and #71 
#### Where should the reviewer start?

Load this branch and run ChromeIDE.
#### How should this be manually tested?

 Create a test file and enter in some test code. Try to use each of the token categories listed in issue 71 to ensure they are being properly colored.
#### Any background context you want to provide?

The highlighter is based on the VB code file. The css theme is modified from neo. 
Only piece not working as it should is the etDirective type.
#### What are the relevant tickets?

Issues #71 and #85 
#### Screenshots (if appropriate)

![image](https://cloud.githubusercontent.com/assets/2586086/7714179/5b9f4bf8-fe31-11e4-85f7-ac6ca0e1205d.png)
#### Questions:
- Is there a blog post?
  No
- Does the knowledge base need an update?
  No
